test/validation/api/crypto/odp_crypto_test_inp.c

```diff
@@ -2483,21 +2560,71 @@ static int crypto_suite_packet_sync_init(void)
 	return 0;
 }
 
-static int crypto_suite_packet_async_init(void)
+static int crypto_suite_packet_async_plain_init(void)
 {
+	odp_queue_t out_queue;
+
 	suite_context.packet = true;
 	suite_context.op_mode = ODP_CRYPTO_ASYNC;
 
 	suite_context.pool = odp_pool_lookup("packet_pool");
 	if (suite_context.pool == ODP_POOL_INVALID)
 		return -1;
 
+	out_queue = plain_compl_queue_create();
+	if (ODP_QUEUE_INVALID == out_queue) {
+		fprintf(stderr, "Crypto outq creation failed.\n");
+		return -1;
+	}
 	suite_context.queue = odp_queue_lookup("crypto-out");
 	if (suite_context.queue == ODP_QUEUE_INVALID)
 		return -1;
+	suite_context.q_type = ODP_QUEUE_TYPE_PLAIN;
+	suite_context.compl_queue_deq = plain_compl_queue_deq;
+
 	return 0;
 }
 
+static int crypto_suite_packet_async_sched_init(void)
+{
+	odp_queue_t out_queue;
+
+	suite_context.packet = true;
+	suite_context.op_mode = ODP_CRYPTO_ASYNC;
+
+	suite_context.pool = odp_pool_lookup("packet_pool");
+	if (suite_context.pool == ODP_POOL_INVALID)
+		return -1;
+
+	out_queue = sched_compl_queue_create();
+	if (ODP_QUEUE_INVALID == out_queue) {
+		fprintf(stderr, "Crypto outq creation failed.\n");
+		return -1;
+	}
+	suite_context.queue = odp_queue_lookup("crypto-out");
```

Comment:
Instead of lookup, just save the out_queue handle here.

Reply:
Ack.